engine/src/cython/engine.cpp

```diff
@@ -276,7 +276,7 @@ TableScanInfo getTableScanInfo(std::string logicalPlan){
 	return TableScanInfo{relational_algebra_steps, table_names, table_columns};
 }
 
-std::pair<std::unique_ptr<PartitionedResultSet>, error_code_t> runQuery_C(int32_t masterIndex,
+std::pair<std::shared_ptr<ral::cache::graph>, error_code_t> runGenerateGraph_C(int32_t masterIndex,
```

Comment:
These changes are due to the C wrappers were merged to branch-0.15 before the update. Since there were changes in the UCX branch (specifically on the run_query function), some conflicts were raised. So, now those conflicts are fixed and the branch compiles ok again.